arbor/backends/gpu/shared_state.hpp

```diff
@@ -40,6 +40,17 @@ struct ARB_ARBOR_API ion_state {
     using solver_type = arb::gpu::diffusion_state<arb_value_type, arb_index_type>;
     using solver_ptr  = std::unique_ptr<solver_type>;
 
+    // ARB_SERDES_ENABLE(ion_state,
+                      // node_index_,
+                      // write_Xi_, write_eX_, write_Xo_,
+                      // iX_, eX_, Xi_, Xd_, Xo_, gX_,
+                      // init_Xi_, init_Xo_, init_eX_,
+                      // reset_Xi_, reset_Xo_,
+                      // charge);
```

Comment:
remove?

Reply:
Yes, there are/were alternatives that allow inspecting a more complete picture of the overall state.